index.html

```diff
@@ -2,16 +2,32 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="stylesheet" href="light.css">
-    <title>First</title>
+    <link rel="stylesheet" href="styles/What.css">
+    <title>Second</title>
   </head>
   <body>
-  <h1>Моя первая страница</h1>
-  <div id="text"><a href="about/index.html">Об авторе</a>
-    <div id="light">
-      <div id="rope">
-      </div>
-    </div>
+  <div>
```

Comment:
You need to add wrapper with class .container
and add properties max-width:1440px
margin: 0 auto
it's allow you align your items to the center